modules/cnv-creating-vm.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * cnv_users_guide/cnv-create-vms.adoc
+
+[id="cnv-creating-vm_{context}"]
+= Creating a virtual machine from the CLI
+
+The `spec` object of the VirtualMachine configuration file references
+the virtual machine settings, such as the number of cores and the amount
+of memory, the disk type, and the volumes to use.
+
+Attach the virtual machine disk to the virtual machine by referencing
+the relevant PVC `claimName` as a volume.
+
+[NOTE]
+====
+https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/[ReplicaSet]
+is not currently supported in {ProductName}.
+====
+
+See the Reference section for information about volume types and
+sample configuration files.
+
+.Domain settings
```

Comment:
Move all of these references to a different module - you can't have this much reference material as the intro to a procedure module.

Reply:
Good point. I moved the actual steps (and numbered them) to the top:

.Procedure

The `spec` object of the VirtualMachine configuration file references
the virtual machine settings, such as the number of cores and the amount
of memory, the disk type, and the volumes to use.

. Attach the virtual machine disk to the virtual machine by referencing
the relevant PVC `claimName` as a volume.

. To create a virtual machine with the {product-title} client, run this command:

----
$ oc create -f <vm.yaml>
----

. Since virtual machines are created in a *Stopped* state, run a virtual machine
instance by starting it.